### ISSUE TITLE
Freezes making reference

### ISSUE THREAD

**Issue by ushasigh** [Enchufa2/video-tester#5]
Where does the measures get saved? And how long does it take to make references?

**Comment by Enchufa2** [comment]
See the docs: http://enchufa2.github.io/video-tester/usage.html

How long, depends on the size of the videos of course.

**Comment by ushasigh** [comment]
Hi, it is a 20s long video but it just does not output anything for hours. 
It is stuck at the "making reference...."

**Comment by Enchufa2** [comment]
No, a 20 s video should be quick. If you upload the video and configuration somewhere, I could try to debug it.

**Comment by ushasigh** [comment]
Hi, you can find the video here and the VT conf file I am using. Essentially, I am running the client from inside a linux network namesapce. https://drive.google.com/drive/folders/1RHlSFpuW9ndULgumZ08uVlgtaclAtP3M?usp=sharing

**Comment by Enchufa2** [comment]
It seems that the reference pipeline doesn't work anymore with mpeg4, not sure why. It does work with other formats, e.g. H264.

**Comment by parthpadhiar** [comment]
Not able to install all libraries "pcap" and not running it What to do?

I am using python3.12 and ubuntu18.04
